.doc/typedapi/examples/getdoc.asciidoc

```diff
@@ -0,0 +1,24 @@
+[[retrieving_document]]
+==== Retrieving a document
+
+Retrieving a document follows the API as part of the argument of the endpoint.
+In order you provide the `index`, the `id` and then run the query:
+[source,go]
+-----
+res, err := es.Get("index-name", "id").Do(context.Background())
+-----
+
+==== Checking for a document existence
+
+If you do not whish to retrieve the content of the document and want only to check if it exists in your index, we provide the `IsSuccess` shortcut:
```

Comment:
typo: wish

Reply:
When your brain cannot decide between which and wish, thanks for spotting this. :sweat_smile: